main.py

```diff
@@ -371,6 +389,11 @@ def perform_TAS():
                     area.djose.temple()
                     area.djose.trials()
                     game.step = 3
+                    if game_vars.create_saves():
+                        import pathing
```

Comment:
This should be moved up with the other imports.

Reply:
This and one other instance of mid-file import have been moved to the top.